ngboost/ngboost.py

```diff
@@ -395,9 +395,24 @@ def feature_importances_(self):
             return None
         # Reshape the base_models
         params_trees = zip(*self.base_models)
+
         # Get the feature_importances_ for all the params and all the trees
+
+        def _get_feature_importance(tree, tree_index):
+            tree_feature_importance = getattr(tree, "feature_importances_")
+            if self.col_sample == 1.:
```

Comment:
Is this condition necessary? I think we can leave it out and it will still work the same.

Reply:
if there was no `col_sample` used then there is no need to do the setting values to zeros array, but I think we can remove it since the cost seems negligible.